content/docs/introduction/roadmap.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Roadmap
+sort_rank: 6
+---
+
+# Roadmap
+
+The following is only a selection of some of the major features we plan to
+implement in the near future. To get a more complete overview of planned
+features and current work, see the issue trackers for the various repositories,
+for example, the [Prometheus
+server](https://github.com/prometheus/prometheus/issues).
+
+**Hierarchical federation**
+
+Hierarchical federation will allow higher-level Prometheus servers to collect
+aggregated time series data from subordinated servers. This will enable more
+scalable monitoring topologies. For example, a setup might consist of
+per-datacenter Prometheus servers that collect data in high detail, and a set
+of global Prometheus servers which collect and store only aggregated data from
+those local servers. This allows you to have an aggregate global view and
+detailed local views.
+
+GitHub issue: [#480](https://github.com/prometheus/prometheus/issues/480)
+
+**Aggregatable histograms**
+
+The current client-side [summary
+types](http://localhost:3000/docs/concepts/metric_types/#summaries) do not
+support aggregation of quantiles. For example, it is [statistically
+incorrect](http://latencytipoftheday.blogspot.de/2014/06/latencytipoftheday-you-cant-average.html)
+to average over the 90th percentile latency of multiple monitored instances.
+We plan to implement server-side histograms which will allow for this use case.
+
+GitHub issue: [#9](https://github.com/prometheus/prometheus/issues/9)
+
+**More flexible label matching in binary operations**
+
+[Binary operations](/docs/querying/operators/) between time series vectors
+currently require exact matches of label sets on both sides of the operation
+in order for paired vector elements to propagate into the result. We plan to
+add more flexible label-matching features to support advanced query use cases.
+
+GitHub issue: [#393](https://github.com/prometheus/prometheus/issues/393)
```

Comment:
It is, but I found it a quite vital one. What do you propose? Renaming the section to "Query language improvements" and listing several examples?